test/plugins/razorpay/utils/mockRazorpay.ts

```
@@ -0,0 +1,349 @@
+    updatedAt: new Date(),
+    ...overrides,
+  };
+};
+
+/**
+ * Mock database client for testing
+ */
+export const createMockDatabaseClient = () => {
+  const mockDb: any = {
+    // Delegate 'then' to 'execute' so that await checks receive the result of execute()
+    then: (resolve: any, reject: any) => mockDb.execute().then(resolve, reject),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)